diff --color -Naur a/src/main.cpp b/src/main.cpp
--- a/src/main.cpp	2022-05-03 00:51:41.396869912 +0100
+++ b/src/main.cpp	2022-05-03 00:54:45.486873118 +0100
@@ -40,24 +40,6 @@

 int main(int argc, char **argv)
 {
-#ifndef Q_OS_WIN
-    // Prohibit using sudo or kdesu (but allow using the root user directly)
-    if (getuid() == 0) {
-        if (!qEnvironmentVariableIsEmpty("SUDO_USER")) {
-            std::cout << "Running Dolphin with sudo can cause bugs and expose you to security vulnerabilities. "
-                         "Instead use Dolphin normally and you will be prompted for elevated privileges when "
-                         "performing file operations that require them."
-                      << std::endl;
-            return EXIT_FAILURE;
-        } else if (!qEnvironmentVariableIsEmpty("KDESU_USER")) {
-            std::cout << "Running Dolphin with kdesu can cause bugs and expose you to security vulnerabilities. "
-                         "Instead use Dolphin normally and you will be prompted for elevated privileges when "
-                         "performing file operations that require them."
-                      << std::endl;
-            return EXIT_FAILURE;
-        }
-    }
-#endif

     /**
      * enable high dpi support
